{{cookiecutter.project_slug}}/Makefile

```diff
@@ -48,12 +50,14 @@ setup: ### Setup remote environment
 		--volume $(PROJECT_PATH_STORAGE):$(PROJECT_PATH_ENV):ro \
 		$(BASE_ENV_NAME) \
 		'sleep 1h'
+	@echo
```

Comment:
needed for testing: somehow no space was printed

Reply:
I don't like having these echos in our production Makefile. Do we really need them?

Reply:
no, i implemented run helpers in a way that we really don't need these echoes. Maria please delete them (also from the rest code)

Reply:
wait: this is another kind of echo: it's needed to add a new line. try to delete or and room tests, of they fail, please fix the test patterns.

Reply:
so other echoes like "echo Success" can be deleted